tracing-attributes/Cargo.toml

```diff
@@ -40,13 +40,16 @@ async-await = []
 [dependencies]
 syn = { version = "1", features = ["full", "extra-traits"] }
 quote = "1"
+proc-macro2 = "1"
 
 
 [dev-dependencies]
 tracing = { path = "../tracing", version = "0.1" }
 tracing-futures = { path = "../tracing-futures", version = "0.2" }
 tokio-test = { version = "0.2.0" }
 tracing-core = { path = "../tracing-core", version = "0.1"}
+# I'm not a huge fan of adding such a dependency only for tests, any idea ?
+async-trait = "0.1"
```

Comment:
You're right, "Dev-dependencies are not used when compiling a package for building, but are used for compiling tests, examples, and benchmarks.

These dependencies are not propagated to other packages which depend on this package."

So it's finally not that big of an issue, as you said. Alas, I never took the time to "read" (instead of looking at it when in need) the cargo book and it looks like I'm still learning the basics of cargo sometimes ^^

Reply:
Great, let's just add the dev dependency, then. Since this isn't an issue, we can remove the comment.